code/modules/mob/living/simple_animal/hostile/venus_human_trap.dm

```diff
@@ -184,17 +202,29 @@
 			if(O.density)
 				return
 
-	var/datum/beam/newVine = Beam(the_target, "vine", maxdistance = vine_grab_distance, beam_type=/obj/effect/ebeam/vine)
-	RegisterSignal(newVine, COMSIG_PARENT_QDELETING, PROC_REF(remove_vine), newVine)
-	vines += newVine
-	if(isliving(the_target))
-		var/mob/living/L = the_target
-		L.Paralyze(20)
+	// Only create a new vine if the target is a mob or an object
+	if(ismob(the_target) || isobj(the_target))
+		var/datum/beam/newVine = Beam(the_target, "vine", maxdistance = vine_grab_distance, beam_type=/obj/effect/ebeam/vine)
+		RegisterSignal(newVine, COMSIG_PARENT_QDELETING, PROC_REF(remove_vine), newVine, override = TRUE)
+		vines += newVine
+
+	var/mob/living/L = the_target
+	if(iscarbon(L))
+		L.apply_damage(30, STAMINA, BODY_ZONE_CHEST)
+		L.Knockdown(3 SECONDS)
+		to_chat(L, span_alert("The vines knock you down"))
```

Comment:
```suggestion
		L.apply_damage(30, STAMINA, BODY_ZONE_CHEST)
		L.Knockdown(3 SECONDS)
		L?.pulledby.stop_pulling()
		to_chat(L, span_alert("The vines knock you down"))
```

This will cause a vines to break grabs so players will have to grab the person again. If this wasn't here, players could simply hold each other to be immune to grabbing.